### ISSUE TITLE
Use cases for sizeByDistortedWh? 

### ISSUE THREAD

**Issue by jpstroop** [IIIF/api#879]
I'm becoming increasingly annoyed by `sizeByDistortedWh` vs `sizeByWh`. Any time you scale down an image in which either dimension is an odd number of pixels, you are distorting the image. Doesn't this mean that it's impossible for a server to say claim that it doesn't support the `sizeByDistortedWh` feature? E.g.:

```
>>> # Given an image that is 6000 wide by 4001 high:
>>> (6000 / 4001)
1.4996250937265683
>>> (3000 / 2000) # half size
1.5
```

The aspect ratio has changed, so it's distorted.

If we're going to change the canonical syntax for size to `w,h` in 3.0, we need to think about this carefully. I'm tempted to propose that we drop this distinction and say that something like:

> Servers MUST return an image that is true to the long dimension of the request, and MAY scale the short dimension as in order to preserve as accurately as possible the aspect ratio implied by the width and height given in the info.json.

Thoughts? Are there real use cases for severely distorting images?

**Comment by zimeon** [comment]
I agree that for most images, most scaling requests are actually distorted even if the intention is simple scaling. Because of this I find it hard to imagine an image server implementation that won't be able to do arbitrarily, including severely, distorted image. So, coming at this from the other side than @jpstroop: While I don't see compelling use cases for severely distorted images, I do see that in various applications it will be easiest for a client to ask for a specific `w,h` than to work out what is gets back from any non-specific request. 

Since this is a 3.0 issue, I wonder whether we can later survey implementations and see whether any claim to support `sizeByWh` without supporting `sizeByDistortedWh`. I suspect not.

**Comment by jpstroop** [comment]
> I do see that in various applications it will be easiest for a client to ask for a specific `w,h` than to work out what is gets back from any non-specific request.

Or spec/warn that servers may redirect to `w,h` that it feels best preserve the aspect ratio (while preserving the long dim, imo)

**Comment by zimeon** [comment]
@IIIF/editors agree that `sizeByWhDistorted` has no useful meaning separate from `sizeByWh`. Will remove `sizeByWhDistorted`. Also change descriptions of aspect ratio preserving requests to being as close as possible to aspect ratio preserving because they will often not be exact.

**Comment by azaroth42** [comment]
Closed by #1424
